### PR TITLE
Use static testdata if present

### PR DESCRIPTION
**Pull request by TheChymera**
We should support static testdata now that we have releases.

### CODE REVIEW

**Comment by TheChymera** [comment]
@effigies --- regarding decreasing coverage, unsure whether we should fully execute this as part of the test suite.

It's kind of a special case for releases. If we were to link it to some specific test data archive, that would actually be expected to sometimes fail between releases as meaningful differences between static archives and the live example repos accumulate.

**Comment by effigies** [comment]
We should exclude the `tests/` directory from coverage in any case. A path not hit there is not indicative of anything.

File: tools/schemacode/pyproject.toml [hunk 1]
```diff
@@ -29,3 +29,8 @@ multi_line_output = 3
 markers = [
   "validate_schema: tests that validate the schema itself",
 ]
+
+[coverage:run]
+parallel = True
+omit =
+    bidsschematools/tests/*py
```
Comment (effigies) [inline tools/schemacode/pyproject.toml:36]: TOML syntax.

```suggestion
parallel = true
omit = ["*/tests/*"]
```

Also, I think we need to update the test dependencies in setup.cfg to use `coverage[toml]` to ensure this gets picked up.

File: tools/schemacode/setup.cfg [hunk 2]
```diff
@@ -36,7 +36,7 @@ render =
     pandas
 tests =
     codecov
-    coverage<5.0
+    coverage[toml]<5.0
     flake8<5.0
```
Comment (effigies) [inline tools/schemacode/setup.cfg:40]: While we're fiddling around here, can we try relaxing these? I suspect they were needed to wait for other tooling to catch up with an API-breaking change, but I hope we won't need them forever.

```suggestion
    coverage[toml]
    flake8
```

**Review by effigies — APPROVED** [review]

**Comment by TheChymera** [comment]
@effigies don't pull yet, codecov is still iffy :/

**Comment by effigies** [comment]
Oh, do we just need to tell codecov to ignore these as well?

https://docs.codecov.com/docs/codecovyml-reference#ignore

**Comment by effigies** [comment]
Sorry, I meant I think we need a `codecov.yml` file in addition to configuring coverage.

**Comment by TheChymera** [comment]
@effigies I have no idea why this works, but there we have it :3

**Comment by TheChymera** [comment]
@effigies no, it doesn't work,,, it does this weird thing where it pretends it worked for a few minutes and then says it didn't -.-

**Comment by TheChymera** [comment]
@effigies ok, this works, project-wide coverage is down not due to the addition of uncovered code, but the removal of valid but irrelevant code from coverage.

**Pull request merged**